package.json

```diff
@@ -83,6 +83,7 @@
     "nps": "^5.7.1",
     "nps-utils": "^1.5.0",
     "path": "^0.12.7",
+    "pg": "^7.4.3",
```

Comment:
Actually `typeorm` is already depends on this library, so I'm not sure whether it needed in this project, since project is using ORM for manage DB.